test/load-generator/boulder-calls.go

```diff
@@ -31,11 +32,12 @@ var (
 	// stringToOperation maps a configured plan action to a function that can
 	// operate on a state/context.
 	stringToOperation = map[string]func(*State, *context) error{
-		"newAccount":    newAccount,
-		"getAccount":    getAccount,
-		"newOrder":      newOrder,
-		"fulfillOrder":  fulfillOrder,
-		"finalizeOrder": finalizeOrder,
+		"newAccount":        newAccount,
+		"getAccount":        getAccount,
+		"newOrder":          newOrder,
+		"fulfillOrder":      fulfillOrder,
+		"finalizeOrder":     finalizeOrder,
+		"revokeCertificate": revokeCertificate,
```

Comment:
Could you also update `test/load-generator/integration-test-config.json` to use `"revokeCertificate"` in the `"plan"` that gets run in CI?

_(You can ignore the pebble load-generator config, I will open a separate PR to delete that config file)_

Reply:
> I will open a separate PR to delete that config file)

:ballot_box_with_check: https://github.com/letsencrypt/boulder/pull/4238